site/docs/components/pagination/accessibility.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Keyboard interactions
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Tab">
+
+- Tab moves focus into the pagination component, setting focus to the first active button.
+- If the first page is active, Tab sets focus on the first page. If any other page is active, sets focus on the **Previous Page** button. If the input field is present and positioned on the left of pagination, initially sets focus in the field.
```

Comment:
This line explains in detail what the first active button is in the line above but reads like it's a completely new point

Reply:
reworded a bit

@mikearildbrown